### ISSUE TITLE
Sending 160+ Characters Long

### ISSUE THREAD

**Issue by amarias-smart** [AdhamAwadhi/JamaaSMPP#49]
Hi, would like to ask for some help. I am pretty new to SMPP is it possible to send a message that is longer that 160 Character. I've been trying to read through the Wiki but can't seem to find articles/documents regarding this. Any help is appreciated. Thanks.

**Comment by mayurrrathod** [comment]
yes, definatly you can send long message using this library without any problem.
